ctapipe/instrument/camera/geometry.py

```diff
@@ -209,25 +209,50 @@ def __getitem__(self, slice_):
             apply_derotation=False,
         )
 
-    @staticmethod
-    def _calc_pixel_area(pix_x, pix_y, pix_type):
+    @classmethod
+    def guess_pixel_area(cls, pix_x, pix_y, pix_type):
         """ recalculate pixel area based on the pixel type and layout
 
         Note this will not work on cameras with varying pixel sizes.
         """
 
-        dist = np.min(np.sqrt((pix_x[1:] - pix_x[0])**2 + (pix_y[1:] - pix_y[0])**2))
+        dist = cls.guess_pixel_width(pix_x, pix_y)
```

Comment:
is this `dist` the radius (?) of the pixel? maybe it could be renamed to something more meaningful

Reply:
Sorry, it's explained in the other methods docstring. It is edge to edge distance for hexagons, edge width for square and diameter for circle